src/lib/metric_ingest.py

```diff
@@ -16,13 +16,15 @@
 from http.client import InvalidURL
 from typing import Dict, List
 
-from lib.context import Context, DynatraceConnectivity
+from lib.context import LoggingContext, Context, DynatraceConnectivity
 from lib.entities.ids import _create_mmh3_hash
 from lib.entities.model import Entity
 from lib.metrics import DISTRIBUTION_VALUE_KEY, Metric, TYPED_VALUE_KEY_MAPPING, GCPService, \
     DimensionValue, IngestLine
 
 UNIT_10TO2PERCENT = "10^2.%"
+MAX_DIMENSION_NAME_LENGTH = 100
+MAX_DIMENSION_VALUE_LENGTH = 250
```

Comment:
I think we should make these overridable with environment variables and add it to table in `HACKING.MD` - I don't really think any customer will need to use it, but that table acts as a documentation of known limits of MINT ingest too.